docs/docs/compatibility-matrix.mdx

```diff
@@ -0,0 +1,19 @@
+---
+id: compatibility-matrix
```

Comment:
I think we should also add to this matrix https://rasa.com/docs/rasa-enterprise/changelog/compatibility-matrix/